### ISSUE TITLE
[Feature Request]: add method: SetVisibilityState/GetVisibilityState to webContents

### ISSUE THREAD

**Issue by replace5** [electron/electron#28322]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a feature request that matches the one I want to file, without success.

### Problem Description

If a window has many webview, the hidden webview should throttling timer like FreezePage to improve performance

### Proposed Solution

add method: SetVisibilityState/GetVisibilityState to webContents, when webview is hidden, SetVisibilityState call SetVisibilityState to Freeze the webview

### Alternatives Considered

-

### Additional Information

-

**Comment by pushkin-** [comment]
seems like a `backgroundThrottling` configuration like the one in [BrowserWindow](https://www.electronjs.org/docs/api/browser-window#page-visibility) would be more appropriate than an API

**Comment by replace5** [comment]
> seems like a `backgroundThrottling` configuration like the one in [BrowserWindow](https://www.electronjs.org/docs/api/browser-window#page-visibility) would be more appropriate than an API

backgroundThrottling run only window  minimize, I want run with webview like chrome tabs

**Comment by pushkin-** [comment]
@replace5 Yeah, I'm not saying to use `backgroundThrottling`. I'm just saying that a configuration for webviews similar to the one for the window is perhaps more appropriate than calling an API to freeze the webcontents.

**Comment by replace5** [comment]
> @replace5 Yeah, I'm not saying to use `backgroundThrottling`. I'm just saying that a configuration for webviews similar to the one for the window is perhaps more appropriate than calling an API to freeze the webcontents.

Yes, When WebView become hidden, freeze the webcontents; when visible, unfreeze
